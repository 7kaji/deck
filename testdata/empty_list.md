# Empty list

- A
-

<!-- {"layout":"title-and-body"} -->
