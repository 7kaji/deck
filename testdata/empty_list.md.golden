[
  {
    "layout": "title-and-body",
    "titles": [
      "Empty list"
    ],
    "bodies": [
      {
        "paragraphs": [
          {
            "fragments": [
              {
                "value": "A"
              }
            ],
            "bullet": "-"
          }
        ]
      }
    ],
    "title_heading_level": 1
  }
]